engineering_tools/carma

```diff
@@ -204,7 +204,26 @@ carma-config__install() {
     echo "Cleaning up temporary container..."
     docker rm carma-config-tmp
 }
+carma__record() {
```

Comment:
I think the name record here is a bit misleading. It implies this command actually causes a recording action to be taken.
I think 
carma fix bag might be more appropriate or something similar.